opensearch/services/datasets/datasets.php

```diff
@@ -351,6 +387,7 @@ public function getQueryBody($type) {
 				$body .= "OPTIONAL { ?dataset dco:associatedDCOPortalGroup ?gp . ?gp rdfs:label ?g_l . } ";
 				$body .= "OPTIONAL { ?project dco:relatedDataset ?dataset ; rdfs:label ?pl . } ";
 				$body .= "OPTIONAL { ?dataset obo:ERO_0000045 ?acc . } ";
+				$body .= "OPTIONAL { ?dataset dco:hasDataType ?d . ?d rdfs:label ?d_l . }";
```

Comment:
Is there a change to the select statement to give us back the datatype uri and it's label?